webpack/assets/javascripts/foreman_tools.js

```diff
@@ -52,7 +52,7 @@ export function activateTooltips(elParam = 'body') {
   el
     .find('*[title]')
     .not('*[rel]')
-    .tooltip({ container: 'body' });
+    .tooltip({ container: 'body', trigger: 'hover' });
```

Comment:
why only on this and not on other tooltip activation methods? might make sense to clean up a bit an unify them